packages/functions/src/oidc/get-vercel-oidc-token.ts

```diff
@@ -23,6 +23,29 @@ import { getContext } from '../get-context';
  * ```
  */
 export async function getVercelOidcToken(): Promise<string> {
+  return getVercelOidcTokenSync();
```

Comment:
can we please just remove the old one, in that case?  it's a noop anyway if someone awaits a sync function, anyway.

so I'm saying instead of creating a new function, can we just update the existing one to not be async?

Reply:
can we update our code snippets to show `() => string | Promise<string>`?  that'd be a backwards compatible change (and we can make that change here too, of course).